src/main/java/com/eppo/sdk/deserializer/BanditsDeserializer.java

```diff
@@ -39,7 +39,7 @@ public Map<String, BanditParameters> deserialize(
             BanditParameters parameters = new BanditParameters();
             parameters.setBanditKey(banditKey);
             parameters.setUpdatedAt(updatedAt);
-            parameters.setModel(model);
+            parameters.setModelName(model);
```

Comment:
I decided `modelName` was a better, more explicit name for this field.

Reply:
Oh yeah, we should probably change it in RAC land too since we can. Let's make the change.